src/components/Level/index.js

```diff
@@ -40,38 +41,44 @@ const Level = () => {
       .toString()
   );
 
-  const [redDinos] = useState(
-    new Array(10).fill(undefined).map(() => {
-      return {
-        Component: OnePointRedDino,
-        style: {
-          transform: `
-            scaleX(-1)
-            translate(
-              ${randBetween(-25, 25)}%,
-              ${randBetween(-100, 100)}%
-            )`,
-        },
-      };
-    })
+  const [redDinos] = useState(() => {
```

Comment:
Maybe we could generate the random dinos here instead of on line 97 and with `dinoPerLevelRange`.
It'd be nice if we could just do this instead and have everything in one array.
```js
{redDinos.map(({Component}) => {
    return <Component />
}}
```
Also, whenever the state changes, the dinos will regenerate this way!

Reply:
Also, we could perhaps shuffle the array so that the same type of dinos aren't always grouped together. Check out this package for an easy way to do it.
https://www.npmjs.com/package/shuffle-array

Reply:
We could put this into a custom hook so we could reuse it for blueDinos as well if you are up to it.
https://reactjs.org/docs/hooks-custom.html

Reply:
Okay so I went back to how you had it originally and tried again. This should be better I hope! I will look into creating a custom hook for how I have it. Just haven't done that part yet. Let me know what you think! Thanks a lot for the guidance.